package.json

```diff
@@ -57,6 +58,5 @@
   "devDependencies": {
     "concurrently": "^5.3.0",
     "prettier": "^2.2.1"
-  },
-  "proxy": "http://localhost:5000"
+  }
```

Comment:
You make me doubt myself...
This change seems to me it will break everything (add only one bracket, without it's counterpart)